VideoLocker/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
     // with module updates.
     compile project (':android-iconify-fontawesome')
     compile fileTree(dir: 'libs', include: '*.jar')
+    compile 'org.apache.commons:commons-lang3:3.4'
```

Comment:
We were already importing version 2 of the Apache Commons Lang library as a transitive dependency of the Commons IO library, and it has lots of convenient utilities. and the `FastDateFormat` in version 3 had increased support for parsing ISO 8601 UTC offsets, so I thought that it would be a net benefit to import and use the latest version of it. Utilities that aren't utilized in the code base would be stripped out by ProGuard anyway. However, per my previous [comment][], we should probably use a dedicated parser for ISO 8601 instead, so I suppose we can leave out this dependency for now.

[comment]: https://github.com/edx/edx-app-android/pull/748#discussion_r67686918